### PR TITLE
[exec](compress) use FragmentTransmissionCompressionCodec control the exchange compress behavior

### PR DESCRIPTION
**Pull request by HappenLee**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/runtime/runtime_state.h [hunk 1]
```diff
@@ -392,9 +392,13 @@ class RuntimeState {
         if (_query_options.__isset.fragment_transmission_compression_codec) {
```
Comment (yiguolei) [inline be/src/runtime/runtime_state.h:392]: should cache the value to avoid check the string value too many times...

**Comment by HappenLee** [comment]
run buildall

File: be/src/runtime/runtime_state.h (hunk 1 above)
Comment (HappenLee) [reply be/src/runtime/runtime_state.h:392]: one query only call once time

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Comment by yiguolei** [comment]
run p0

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**